src/richText.js

```diff
@@ -103,7 +102,18 @@ define([
         }
     });
 
-    CKEDITOR.config.allowedContent = true;
+    CKEDITOR.config.allowedContent = {
+        span: {
+            classes: 'label,label-*',
+            attributes: 'data-output-value,data-value,data-date-format',
+        },
+        i: {
+            classes: 'fcc,fcc-*,fa,fa-*',
+        },
+        output: {
+            attributes: 'value',
+        },
```

Comment:
I'm not sure if we support it, but [some flavors of markdown](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#html) support inline HTML.